client/packages/common/src/intl/locales/ar/common.json

```diff
@@ -2195,7 +2195,6 @@
   "placeholder.search-by-name": "البحث حسب الاسم",
   "placeholder.search-by-name-or-code": "البحث حسب الاسم أو الرمز",
   "placeholder.search-by-notes": "البحث حسب الملاحظات",
-  "preferences": "التفضيلات",
```

Comment:
Cleared this in favour of "Global Preferences" - store prefs will be managed separately, from the facilities list view